cilium/cmd/policy_selectors.go

```diff
@@ -31,10 +35,22 @@ var policyCacheGetCmd = &cobra.Command{
 			sort.Slice(resp, func(i, j int) bool {
 				return resp[i].Selector < resp[j].Selector
 			})
-			fmt.Fprintf(w, "SELECTOR\tUSERS\tIDENTITIES\n")
+			fmt.Fprintf(w, "SELECTOR\tLABELS\tUSERS\tIDENTITIES\n")
```

Comment:
Do we want to print this header if `verbosePolicySelectors` is false?

Reply:
I think it's still useful to have the header if the full labels (verbose) are not shown. In fact, I think the output would be very hard to parse without the headers describing what is what.